ci-operator/config/openshift/release/openshift-release-master__nightly-4.19.yaml

```diff
@@ -615,6 +615,14 @@ tests:
       enable:
       - observers-resource-watch
     workflow: openshift-e2e-vsphere-host-groups-csi
+- as: e2e-vsphere-ovn-multi-vcenter-csi
+  cron: '@weekly'
+  steps:
+    cluster_profile: vsphere-elastic
+    observers:
+      enable:
+      - observers-resource-watch
+    workflow: openshift-e2e-vsphere-multi-vcenter-csi
```

Comment:
I am not seeing the workflow specify multiple vcenters.  You should set POOLS in the env for the job such as:
```
   env:
      POOLS: vcenter.ci.ibmc.devcluster.openshift.com-cidatacenter-cicluster vcenter.ci.ibmc.devcluster.openshift.com-cidatacenter-1-cicluster-1
        vcenter-1.ci.ibmc.devcluster.openshift.com-cidatacenter-2-cicluster-3
```

Reply:
Oh I missed that, thanks a lot for the correction, updated